src/NServiceBus.Core/Transports/Msmq/CorrelationIdMutatorForBackwardsCompatibilityWithV3.cs

```diff
@@ -1,28 +0,0 @@
-﻿namespace NServiceBus.Transports.Msmq
-{
-    using System;
-    using MessageMutator;
-    using Unicast.Messages;
-
-    [ObsoleteEx(RemoveInVersion ="6.0")]
-    class CorrelationIdMutatorForBackwardsCompatibilityWithV3 : IMutateOutgoingTransportMessages
```

Comment:
I say we re-evaluate in 1 years time, so bump it to v7.
By then hopefully not many customers are still in v3 land.